package/boot/uboot-mvebu/patches/136-mmc-xenon-set-signal-voltage-and-max-base-clock.patch

```diff
@@ -0,0 +1,165 @@
+From 7d1f9e4c4c27f9022da04a93cb323a32eec41dfa Mon Sep 17 00:00:00 2001
+From: Evan Wang <xswang@marvell.com>
+Date: Wed, 19 Aug 2020 16:19:39 +0200
+Subject: [PATCH] mmc: xenon: set signal voltage and max base clock
```

Comment:
Can You test this commit https://github.com/tmn505/u-boot/commit/e8380f7db3ab6e8542087963c8297d002976349e, without this patch applied? Last time I booted my board the SD card functioned properly without additional changes to the driver.

Reply:
All the patches here are most likely to be applied for u-boot/next. So they can all be dropped once uboot-mvebu is bumped to v2021.01.

Reply:
That patch you linked is for emmc, right? I have some more patches to that, I'll add those tomorrow.

Reply:
Nope, sdhci0 is not emmc :) That patch doesn't apply, but mainline has a similar setup, so that should be covered.

Reply:
No, that patch is only for SD card, You should apply it without the patches modifying the xenon driver and adding sdcard bindings to dts.

Reply:
This patch was merged to u-boot/master yesterday, I guess that resolves this?

Reply:
@tmn505: If you have u-boot patches for Armada 37xx, why do not you send them to u-boot mailing list for review and inclusion into mainline u-boot? I think it does not make sense to having private fork, like Marvell did.

Reply:
This tree (ebin branch) was specifically intended to send upstream, after adding my SoB and changelog what I did, but only after comphy patches would be accepted. I didn't want to invest time into rebasing it.

@dhewg Yes.

Reply:
I did not know about your fork when I prepared u-boot changes for Armada 3720. Otherwise I would have looked at it. And because I have not seen anything on u-boot list I prepared my own series of patches and sent them.

Reply:
@pali I'm in no way bothered by it, I'm glad You did it.